### ISSUE TITLE
Sort order should not be boolean with latest restapi 

### ISSUE THREAD

**Issue by nzambello** [plone/volto#2397]
**Describe the bug**
With latest versions of plone.restapi, the querystring-search sort order index has changed.

So, if we were using `sort_order: true` or `sort_order: true`, now we should query for `sort_order: descending` or `sort_order: ascending`.

**To Reproduce**
Just create a listing block and set sorting order.
Easy to see using sortable title as `sort_on`

**Expected behavior**
Using a listing block ordered by title, in Edit mode when selecting "Reverse order" you should see contents with titles starting with Z or W, while finding the ones starting with A or B deselecting "Reverse order".

**Software (please complete the following information):**
 - Volto 12.14.0
 - Plone 5.2.3
 - Plone REST API 7.3.5

**Comment by nzambello** [comment]
Implementing a solution (WIP)

**Comment by tisto** [comment]
@nzambello maybe plone.restapi should issue a warning or so when we pass parameters that do not work as expected. We could create an issue on plone.restapi and write a test to prove the behavior, then think about the best way plone.restapi could handle this. Just an idea...

**Comment by nzambello** [comment]
@tisto I think the restapi team should notify the volto team if something that can affect volto changes.
Otherwise, we should keep track of restapi changes and verify every release to work with volto having a "tested version" pinned for volto.

We are insiders and we fixed it fastly (and it's still annoying because we discovered this in production) but it is a big issue for a plone developer who's not so in the core community.
